db/schema.rb

```diff
@@ -26,6 +26,13 @@
     t.index ["user_id"], name: "index_gigs_on_user_id"
   end
 
+  create_table "join_student_skills", force: :cascade do |t|
```

Comment:
syntax problem here no?
student is singular vs. skills plural

Reply:
one should be singular and the other plural.
I dropped the term 'join' from the table name.